apps/src/studio/Item.js

```diff
@@ -347,9 +427,15 @@ Item.prototype.display = function () {
 
 Item.prototype.getNextPosition = function () {
   var unit = Direction.getUnitVector(this.dir);
+  var speed = this.speed;
+  // TODO: Better concept of which actions actually move the actor
+  // Projected position should not be in front of you if you are not moving!
+  if (this.activity === "none" || this.activity === "watchActor") {
+    speed = 0;
```

Comment:
:+1: Seems good. Does this mean that previously items could have activity === 'none' and still move around? Did it just rely on Direction.NONE? (dir === Direction.NONE and speed === 0 are perhaps redundant)